sdk/lib/pathcch/pathcch.h

```diff
@@ -1,360 +0,0 @@
-/*
- * PROJECT:     ReactOS PSDK
- * LICENSE:     MIT (https://spdx.org/licenses/MIT)
- * PURPOSE:     "Secure" shell path manipulation functions
- * COPYRIGHT:   MinGW-64 and Microsoft Corporation.
- */
-
-/**
- * This file is part of the mingw-w64 runtime package.
- * No warranty is given; refer to the file DISCLAIMER within this package.
- */
-
-#pragma once
-
-
-#ifndef WINBASEAPI
-#ifndef _KERNEL32_
-#define WINBASEAPI DECLSPEC_IMPORT
-#else
-#define WINBASEAPI
-#endif
-#endif
-
-
-#ifndef WINPATHCCHAPI
-#ifndef STATIC_PATHCCH
-#define WINPATHCCHAPI WINBASEAPI
-#else
-#define WINPATHCCHAPI
-#endif
-#endif
-
-
-#ifdef __cplusplus
-extern "C" {
-#endif
-
-// typedef enum PATHCCH_OPTIONS
```

Comment:
Could you please move instead this header in the psdk instead? Thanks. I took time to made this header to be complete and compatible with the official MS header, not with an incomplete header like wine's.
You may have to re-enable this typedef (+ convert these defines to enum values) and reenable the DEFINE_ENUM_FLAG_OPERATORS below, to make it fully compatible with MS.
And if wine's kernelbase complains that it cannot use this header for whatever (wrong) reason, then put wine's pathcch.h into the include/reactos/wine subdirectory and make kernelbase use that one instead.....